docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -174,12 +174,26 @@ BASIC_AUTH_USERNAME="" BASIC_AUTH_PASSWORD="" docker compose up
 
 Note: [basic auth](https://docs.airbyte.com/operator-guides/security#network-security) must be disabled by setting `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD` to empty values, otherwise requests from the development server will fail against the local API.
 
-- Start up the react app.
+- Install [`nvm`](https://github.com/nvm-sh/nvm) (Node Version Manager) if not installed
+- Use `nvm` to install the required node version:
 
 ```bash
 cd airbyte-webapp
-npm install
-npm start
+nvm install
+```
+
+- Install the `pnpm` package manager in the required version:
+
+```bash
+# <version> must be the exact version from airbyte-webapp/package.json > engines.pnpm
```

Comment:
Not a blocker for this PR, but it looks like pnpm ships with a node version manager too, should we just use that one? One tool less

https://pnpm.io/cli/env

Didn't look into it in detail though

Reply:
I have looked a bit into it, but currently didn't saw much of a benefit over keep using `nvm` (also we anyway need `npm` first to install `pnpm`, so we either need to guide to `nvm` or a local node installation in our docs).

Reply:
You can install pnpm without npm, right? https://pnpm.io/installation